scalafix/input-0_14/src/main/scala/fix/v0_14_0/FixAvroCoder8.scala

```diff
@@ -7,5 +7,6 @@ import com.spotify.scio.coders.Coder
 import org.apache.avro.generic.GenericRecord
 
 object FixAvroCoder8 {
-  val c = Coder[GenericRecord]
```

Comment:
This was working before because `Coder.fallback` was used. Such code should fail after 0.14 update